src/patternfly/components/Alert/examples/alert-danger-example.hbs

```diff
@@ -1,33 +1,44 @@
-{{#> alert pf-c-alert--Modifier="pf-m-danger" pf-c-alert--AriaLabel="Danger Notification"}}
-<div aria-hidden="true" class="pf-c-alert__icon">
-  <i class="fas fa-exclamation-circle"></i>
-</div>
-<div class="pf-c-alert__body">
-  <h4 class="pf-c-alert__title">
-    <span class="pf-u-sr-only">Danger: </span> Danger notification title</h4>
+{{#> alert pf-c-alert--modifier="pf-m-danger" pf-c-alert--attribute='aria-label="Danger Notification"'}}
+  {{#> alert-icon pf-c-alert-icon--danger="true"}}
+  {{/alert-icon}}
+    {{#> alert-body}}
+      {{#> alert-title}}
+        {{#> sr sr--modifier="pf-u-sr-only"}}
+          Danger:
+        {{/sr}}
+        Danger notification title
+      {{/alert-title}}
     <p>This is a description of the notification content.</p>
```

Comment:
can we add in an description hbs for this?

Reply:
done